### ISSUE TITLE
Regression in handling removing existing quota in API Manager

### ISSUE THREAD

**Issue by gcornacchia** [Axway-API-Management-Plus/apim-cli#434]
### APIM-CLI version

1.14.2

### API-Management version

7.7.20211130

### Bug description

With latest version 1.14.2 I've found a regression in handling the removing of existing quotas in API Manager in these 2 scenarios:

1) Application quota
If an existing application is changed by removing a quota restriction in config file, this will not be removed from API Manager.
Consider the same scenario described in #382. This problem already had to be solved in a previous release.

2) System/Application Default Quota on all API:

if I launch the following script

apim.sh settings import -type globalquotas ...

with a json config file that doesn't contain anymore the system/application quota, will not be removed from API Manager even if quotaMode=replace.

**Comment by rathnapandi** [comment]
Hi @gcornacchia, 

Nothing changed related to quota. Can you share the sample configuration to replicate the issues?

**Comment by gcornacchia** [comment]
HI @rathnapandi,
for application quotas the bug occurs when you try to remove all the existing quotas (see steps below).


### step 1) import an application like this:

```
{
  "name": "Test app",
  "state": "approved",
  "enabled": true,
  "organization": "Community",
  "credentials": [
  ],
  "appQuota": {
	    "restrictions": [
      {  "api": "*",
        "method": "*",
        "type": "throttle",
        "config": {
          "messages": "1000",
          "period": "hour",
          "per": "1"
        } },
{  "api": "*",
        "method": "*",
        "type": "throttle",
        "config": {
          "messages": "1000",
          "period": "hour",
          "per": "1"
        } }
    ]
  }
}
```

2 application quotas are created.


### Step 2 -  I want to remove all quotas


```
{
  "name": "Test app",
  "state": "approved",
  "enabled": true,
  "organization": "Community",
  "credentials": [
  ],
  "appQuota": {
	    "restrictions": [
    ]
  }
}
```

No changes detected - quotas are not removed.

**Comment by rathnapandi** [comment]
Hi @gcornacchia 
Empty Quota updates are not handled before 
Can you try testing with  
[axway-apimcli-1.14.3-SNAPSHOT.tar.gz](https://github.com/Axway-API-Management-Plus/apim-cli/files/12623604/axway-apimcli-1.14.3-SNAPSHOT.tar.gz)

This will handle #438 too

**Comment by gcornacchia** [comment]
yes I will test and I will let you know!

**Comment by lturricchia** [comment]
Hi about this topic on removing existing quotas from an APPLICATION, we checked and it's fixed as expected from Apim-cli 1.14.3-snapshot, thanks

we still see a problem in removing GLOBAL quotas for ALL API, as in https://github.com/Axway-API-Management-Plus/apim-cli/issues/442
